app-frontend/packs/components/homePage/header/createButton/CreateButton.vue

```diff
@@ -0,0 +1,22 @@
+<template>
+    <button class="create-btn btn" type="button">Create new banner</button>
+</template>
+
+<script>
+</script>
+
+<style>
+    .create-btn {
+        font-size: 14px;
+        font-weight: 600;
+        color: var(--color-black);
+        background-color: var(--color-white);
+        border-radius: 25px;
+        padding: 16px 24px;
+    }
+
+    .create-btn:hover {
+        box-shadow: 0px 5px 7px #919191;
+        border: .2px solid #b3b3b3;
```

Comment:
У тебя цвета где-то записаны в ловер кейсе - где-то капсом

Лучше привести к одному виду

```suggestion
        border: .2px solid #B3B3B3;
```

Reply:
а ше краще виділить змінні для основних кольорів і юзать їх

Reply:
+, done